pkg/cmd/server/api/types.go

```diff
@@ -545,6 +545,8 @@ type IdentityProvider struct {
 	UseAsLogin bool
 	// MappingMethod determines how identities from this provider are mapped to users
 	MappingMethod string
+	// MappingMethodConfig is used to configure custom MappingMethods.
```

Comment:
How about: "MappingMethodConfig is a list of simple key/value pairs used to configure third-party mappingMethods."